<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Maps">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actionNothing_yet"/>
    </widget>
    <addaction name="actionFile_New_File"/>
    <addaction name="actionFile_Open"/>
    <addaction name="menuRecent_Maps"/>
    <addaction name="actionFile_Save"/>
    <addaction name="actionFile_Save_as"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Import_Maps"/>
    <addaction name="actionFile_Export_Map"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTools">
     <property name="title">
      <string>Tools</string>
     </property>
     <addaction name="actionTools_Paint"/>
     <addaction name="actionTools_Select"/>
     <addaction name="actionTools_Erase"/>
    </widget>
    <widget class="QMenu" name="menuGo_to">
     <property name="title">
      <string>Navigate</string>
     </property>
     <addaction name="actionEdit_Previous_Map"/>
     <addaction name="actionEdit_Next_Map"/>
     <addaction name="separator"/>
     <addaction name="actionEdit_First_Map"/>
     <addaction name="actionEdit_Last_Map"/>
     <addaction name="separator"/>
     <addaction name="actionEdit_Goto_Map"/>
    </widget>
    <addaction name="actionEdit_ResizeMap"/>
    <addaction name="menuGo_to"/>
    <addaction name="actionEdit_Test_Map"/>
    <addaction name="actionEdit_Rename_Map"/>
    <addaction name="menuTools"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Add_Map"/>
    <addaction name="actionEdit_Move_Map"/>
    <addaction name="actionEdit_Insert_Map"/>
    <addaction name="actionClear_Map"/>
    <addaction name="actionEdit_Delete_Map"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Grid"/>
    <addaction name="actionView_Animate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_About"/>
    <addaction name="actionHelp_About_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFile_Open">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/open.gif</normaloff>:/data/icons/open.gif</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open an existing map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/save.gif</normaloff>:/data/icons/save.gif</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save map to disk.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFile_Save_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="statusTip">
    <string>Save map under a different name.</string>
   </property>
  </action>
  <action name="actionFile_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFile_New_File">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/new.gif</normaloff>:/data/icons/new.gif</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="statusTip">
    <string>Create a new map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit_ResizeMap">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/options.gif</normaloff>:/data/icons/options.gif</iconset>
   </property>
   <property name="text">
    <string>Resize Map</string>
   </property>
   <property name="statusTip">
    <string>Resize current map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNothing_yet">
   <property name="text">
    <string>Nothing yet</string>
   </property>
  </action>
  <action name="actionView_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="statusTip">
    <string>Show tile grid on map.</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionTools_Paint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/strawberrypntbrush.png</normaloff>:/data/icons/strawberrypntbrush.png</iconset>
   </property>
   <property name="text">
    <string>Paint</string>
   </property>
   <property name="statusTip">
    <string>Paint Tile on the map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionTools_Select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/colorpicker.png</normaloff>:/data/icons/colorpicker.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="statusTip">
    <string>Select Tile on the map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionTools_Erase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/icons8-erase-40.png</normaloff>:/data/icons/icons8-erase-40.png</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
   <property name="statusTip">
    <string>Erase Tile on the map.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionClear_Map">
   <property name="text">
    <string>Clear Map</string>
   </property>
   <property name="statusTip">
    <string>Clear entire map.</string>
   </property>
  </action>
  <action name="actionHelp_About_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionHelp_About">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About mapEdit</string>
   </property>
  </action>
  <action name="actionEdit_Previous_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/prev.gif</normaloff>:/data/icons/prev.gif</iconset>
   </property>
   <property name="text">
    <string>Previous Map</string>
   </property>
   <property name="statusTip">
    <string>Move to Previous Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionEdit_Next_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/next.gif</normaloff>:/data/icons/next.gif</iconset>
   </property>
   <property name="text">
    <string>Next Map</string>
   </property>
   <property name="statusTip">
    <string>Move to Next Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionEdit_Add_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/plus.png</normaloff>:/data/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Map</string>
   </property>
   <property name="statusTip">
    <string>Add a new Map to this file.</string>
   </property>
  </action>
  <action name="actionEdit_Delete_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/minus.png</normaloff>:/data/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Map</string>
   </property>
   <property name="statusTip">
    <string>Delete Current Map</string>
   </property>
  </action>
  <action name="actionEdit_Insert_Map">
   <property name="text">
    <string>Insert Map</string>
   </property>
   <property name="statusTip">
    <string>Insert a Map at the current index.</string>
   </property>
  </action>
  <action name="actionEdit_Move_Map">
   <property name="text">
    <string>Move Map</string>
   </property>
   <property name="statusTip">
    <string>Move current map to a different position.</string>
   </property>
  </action>
  <action name="actionEdit_Goto_Map">
   <property name="text">
    <string>Go to Map</string>
   </property>
   <property name="statusTip">
    <string>Go to another map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionEdit_Test_Map">
   <property name="text">
    <string>Test Map</string>
   </property>
   <property name="statusTip">
    <string>Test the current map in the emulator.</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFile_Import_Maps">
   <property name="text">
    <string>Import Maps...</string>
   </property>
   <property name="statusTip">
    <string>Import map into the current file.</string>
   </property>
  </action>
  <action name="actionFile_Export_Map">
   <property name="text">
    <string>Export Map...</string>
   </property>
   <property name="statusTip">
    <string>Export the current map.</string>
   </property>
  </action>
  <action name="actionView_Animate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animate</string>
   </property>
   <property name="statusTip">
    <string>Animate the mapview.</string>
   </property>
  </action>
  <action name="actionEdit_Rename_Map">
   <property name="text">
    <string>Rename Map</string>
   </property>
   <property name="statusTip">
    <string>Change the name of this map.</string>
   </property>
  </action>
  <action name="actionEdit_Last_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/go-last.png</normaloff>:/data/icons/go-last.png</iconset>
   </property>
   <property name="text">
    <string>Last Map</string>
   </property>
  </action>
  <action name="actionEdit_First_Map">
   <property name="icon">
    <iconset resource="mapedit.qrc">
     <normaloff>:/data/icons/go-first.png</normaloff>:/data/icons/go-first.png</iconset>
   </property>
   <property name="text">
    <string>First Map</string>
   </property>
  </action>
  <action name="actionResize_Map">
   <property name="text">
    <string>Resize Map</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mapedit.qrc"/>
 </resources>
 <connections/>
</ui>
